app/views/organizations/new.html.erb

```diff
@@ -31,6 +31,16 @@
         <div id="external-type-error-target"></div>
       </div>
 
+      <div class="form-group">
+        <%= f.label "Programs" %>
+        <%= f.select :program_ids,
+                     options_for_select(Program.alphabetical.pluck(:name, :id), @organization.program_ids),
+                     {},
+                     { multiple: true, class: "program form-control", data: { guard: "required", guard_required_target: "#programs-error-target" } } %>
+
+        <div id="programs-error-target"></div>
```

Comment:
same code twice? same as above. Make `views/shared/components/_programs_dropdown.html.erb`?

Reply:
I would rather refactor organizations new/edit views to use the same common form code... but I made a lot of changes in the netsuite branch, so I'd rather put that off until after the netsuite branch is ready to merge in.